apps/server/src/modules/deletion/entity/deletion-request.entity.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { setupEntities } from '@shared/testing';
+import { ObjectId } from '@mikro-orm/mongodb';
+import { DeletionRequestEntity } from '@src/modules/deletion/entity/deletion-request.entity';
+import { DeletionDomainModel } from '../domain/types/deletion-domain-model.enum';
+import { DeletionStatusModel } from '../domain/types/deletion-status-model.enum';
+
+describe(DeletionRequestEntity.name, () => {
+	beforeAll(async () => {
+		await setupEntities();
+	});
+
+	const setup = () => {
+		jest.clearAllMocks();
```

Comment:
```suggestion
		jest.clearAllMocks();
```
Please move it to beforeEach()

Reply:
done